src/index.html

```diff
@@ -4,9 +4,430 @@
+              <a href="#video" class="nav__link">
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div class="main__wrapper wrapper">
+        <div class="card" data-qa="card">
+          <div class="product card__dims">
+            <img
+              src="./images/imac.jpeg"
+              alt="iMac image"
+              class="product__image"
+            >
+            <div class="product__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+
+            <span class="product__info">Product code: 195434</span>
+
+            <div class="product__review">
+              <span class="stars stars--4">
+                <span class="stars__star"></span>
+
+                <span class="stars__star"></span>
+
+                <span class="stars__star"></span>
+
+                <span class="stars__star"></span>
+
+                <span class="stars__star"></span>
+              </span>
+              <span class="product__review-style">Reviews: 5</span>
+            </div>
+
+            <div class="product__price">
+              <span class="product__price-desc">Price:</span>
+
+              <span class="product__price-value">$2,199</span>
+            </div>
+
+            <div class="product__button">
+              <a
+                class="product__button-link"
+                data-qa="card-hover"
+                href="#"
+              >
+                Buy
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="card">
+          <div class="product card__dims">
+            <img
+              src="./images/imac.jpeg"
+              alt="iMac image"
+              class="product__image"
+            >
+            <div class="product__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+
+            <span class="product__info">Product code: 195434</span>
+
+            <div class="product__review">
+              <span class="stars stars--4">
+                <span class="stars__star"></span>
+
+                <span class="stars__star"></span>
+
+                <span class="stars__star"></span>
+
+                <span class="stars__star"></span>
+
+                <span class="stars__star"></span>
+              </span>
+              <span class="product__review-style">Reviews: 5</span>
+            </div>
+
+            <div class="product__price">
+              <span class="product__price-desc">Price:</span>
+
+              <span class="product__price-value">$2,199</span>
+            </div>
+
+            <div class="product__button">
+              <a
+                class="product__button-link"
+                data-qa="hover"
+                href="#"
+              >
+                Buy
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="card">
+          <div class="product card__dims">
```

Comment:
I don't think you need to add `product` alongside `card`, it could be just `card`